### PR TITLE
raw_disk: move packages to manifest

### LINKED ISSUE #616: Consume manifest.yaml in image builder script
**Is your feature request related to a problem? Please describe.**

Make use of `manifest.yaml` file to list all the packages needed for the image builder script.

**Describe the solution you'd like**

Consider suggestions form https://github.com/rancher-sandbox/cOS-toolkit/pull/608#discussion_r702968349

This way we can think of disk customization as we do with ISOs.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Itxaka**
Closes #616 

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

**Comment by Itxaka** [comment]
Not sure about the format

This way its expandable in the future if we need more parameters per package, but on the other side its more complicated than a list of packages...

**Review by mudler — APPROVED** [review]
Like it!

File: manifest.yaml [hunk 1]
```diff
@@ -8,6 +8,24 @@ packages:
   - live/grub2-efi-image
   - recovery/cos-img
 
+# Raw disk creation values start
+
+# Where to get the packages from
```
Comment (mudler) [inline manifest.yaml:13]: maybe we can have as part of a general `raw_disk` section?

Comment (Itxaka) [reply manifest.yaml:13]: works for me

**Pull request merged**